MemberExpectations.md

```diff
@@ -0,0 +1,59 @@
+their individual opinions during discussions within the organization. All
+contributors are expected to be respectful of these different viewpoints,
+and to work collaboratively with one another in a manner that constructively
+elevates all contributors.
+
+When an individual contributor chooses to accept an invitation to participate
+in a leadershp role, they implicitly take on an additional obligation to protect
+and be respectful of both the project and all other contributors.  When
+decisions are made within the established guidelines and policies
+of the project, those in leadership roles have a responsibility to uphold
+and respect the decision even if they disagree with it. This is especially
+important in external communications, for example in social media. Should
+the member be unwilling or unable to do so, then they should step down and
+resign their leadership position. This does not mean that decisions cannot
+be revisited and discussed within the team at a later time.
+
+Members of our leadership groups must also conduct themselves in a
+professional and respectful manner. Some general guidelines include:
+
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes - however they should act swiftly to
+  acknowledge mistakes and correct them. Most often this can mitigate
+  any damage done.
+- Aim to remediate first and then discuss.  If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
+- Treat all community members with respect, consideration, and highest
+  standards of ethical conduct.  
+- Valuing a diversity of views and opinions. Strive to avoid preferential
+  treatment, and hold everyone (including ourselves) accountable to the same
+  set of standards.  Everyone gets to speak up. 
+- Deal with issues directly with the person in question. Resist complaining
+  about others in the project in a public sphere.
+- Build trust by keeping your promises.
+- Be the model of accountability and leadership. Provide the example of
+  ownership and stewardship that everyone can follow to success.
+- Commit to ongoing development and learning best practices for governing.
+- Critique ideas rather than individuals, discussing any concerns in person
+  whenever possible, and taking responsibility for our statements by
+  speaking as much as possible
+  in the first person (.I. statements) rather than in the third person.
+  
+While the guidelines above focus primarily on the spaces where
+we participate in official foundation work (github, irc, conferences),
+it is important to recognize that the public behavior of members also
+reflects on the Node.js project.
+
+If it appears that any member of the project leadership is acting outside
+of the expectations set above please refer to our moderation policy which
```

Comment:
Nit: link to the moderation policy

More than a nit: Does this make sense? The moderation policy is where certain violations should be directed, but does it make sense for violations of *these* expectations? These aren't even a policy.  This document is a guideline, right?

Reply:
added link, and yes we want people to be able to hold people accountable to these expectations.